lib/magic_curl/bin/curl

```diff
@@ -5,12 +5,18 @@ cache_dir=$(sed '/^\#/d' $(dirname ${0})/../conf/cache.conf | grep 'cache_dir'
 
 # find the real curl cmd
 for c in $(which -a curl); do
-  if [ ${c} != ${0} ]; then
+  if [ "${c}" != "${0}" ]; then
     real_curl=${c}
     break
   fi
 done
 
+if [ -z "$real_curl" ]; then
+  echo "curl is not on this system"
+  echo "curl is not on this system" >&2
```

Comment:
not exactly sure why we're echoing to both streams, but here's a slightly DRY'er version:

```
echo "curl is not on this system" | tee /dev/stderr 
```

Reply:
I wanted to make sure it ends up in anywhere a dev would look if they're debugging

Reply:
I updated with your version instead.